content/en/docs/setup/install/multicluster/single-network/primary/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Add a Primary Cluster
+description: Describes how to add a primary cluster to your mesh on a flat network.
+weight: 20
+keywords: [multicluster, primary-cluster, deployment]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, you can add
+{{< gloss "primary cluster" >}}primary clusters{{< /gloss >}} to your
+{{< gloss >}}service mesh{{< /gloss >}}.
+
+To give you clean instructions, this section starts right after you complete the
+[initial configuration](/docs/setup/install/multicluster/single-network/initial-configuration),
+and assumes that `Cluster_1` and `Cluster_2` reside on the same network.
+However, you can add primary clusters to your existing mesh regardless of your
+current [deployment model](/docs/ops/deployment/deployment-models/).
+
+Add primary clusters to your deployment to improve the availability of your
+mesh. To keep multiple {{< gloss "control plane" >}}control planes{{< /gloss >}}
+in sync, apply your configuration to each cluster. Production systems employ
+continuous integration and continuous deployment (CI/CD) pipelines to apply
+configurations since they typically have at least one control plane per region.
+Administrators use these pipelines to manage configuration rollouts. Apply new
+configurations to one primary cluster at a time to help minimize
+troubleshooting.
+
+The following diagram shows a multicluster deployment with two primary
+clusters:
+
+{{< image width="75%"
+    link="primary.svg"
+    caption="A multicluster deployment with two primary clusters"
+    >}}
+
+Complete the [initial configuration instructions](/docs/setup/install/multicluster/single-network/initial-configuration)
+before you continue.
+
+## Configure Trust
+
+{{< boilerplate trust-config >}}
+
+## Deploy Istio
+
+Next, deploy a full Istio control plane on `Cluster_2`. The new control plane
+requires the following configurations to enable a multicluster deployment:
+
+<table>
+    <thead>
+        <tr>
+            <th>Configuration field</th>
+            <th>Description</th>
+            <th>Value</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>clusterName</code>
+            </td>
+            <td>Specifies a human-readable cluster name.</td>
+            <td><code>${CLUSTER_2}</code></td>
+        </tr>
+        <tr>
+            <td><code>network</code></td>
+            <td>Specifies a network ID as an arbitrary string. All clusters in your mesh
+            must be on the same network, and have the same network ID.</td>
+            <td><code>${NETWORK_1}</code></td>
```

Comment:
If there's only one network, how about ${NETWORK} throughout, without the _1?

Reply:
The naming schema enables content reusability.